src/cbmc/cbmc_parse_options.cpp

```diff
@@ -52,7 +52,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <goto-programs/string_instrumentation.h>
 
 #include <goto-symex/rewrite_union.h>
-#include <goto-symex/adjust_float_expressions.h>
+#include <goto-programs/adjust_float_expressions.h>
```

Comment:
This belongs in the group one above now. (Well, `rewrite_union` should likely be moved to `goto-programs` as well.)